skimage/transform/radon_transform.py

```diff
@@ -67,6 +67,11 @@ def radon(image, theta=None, circle=True, preserve_range=False):
     if theta is None:
         theta = np.arange(180)
 
+    if preserve_range is None and np.issubdtype(image.dtype, np.integer):
+        warn('Image dtype is not float. To keep old behavior, '
```

Comment:
This needs to be fleshed out a little more.

> Imge dtype is not float. By default radon will assume you want to preserve the range of your image (preserve_range=True). In scikit-image 0.18 this behavior will change to preserve_range=False. To avoid this warning, explictiely specify the preserve_range parameter.

Reply:
Done